File: client/src/components/HelloAlgolia.js
// hello_algolia.js
const algoliasearch = require("algoliasearch");

// Connect and authenticate with your Algolia app
const client = algoliasearch("O2VY3SXRXO", "37436eb3cc315d090dcde25c9dc06ec2");

// Create a new index and add a record
const index = client.initIndex("test_index");
const record = { objectID: 1, name: "test_record" };
index.saveObject(record).wait();

// Search the index and print the results
index.search("test_record").then(({ hits }) => console.log(hits[0]));

File: server/hello_algolia.py
# hello_algolia.py
from algoliasearch.search_client import SearchClient

# Connect and authenticate with your Algolia app
client = SearchClient.create("O2VY3SXRXO", "37436eb3cc315d090dcde25c9dc06ec2")

# Create a new index and add a record
index = client.init_index("test_index")
record = {"objectID": 1, "name": "test_record"}
index.save_object(record).wait()

# Search the index and print the results
results = index.search("test_record")
print(results["hits"][0])
